tests/unit/plugins/module_utils/test_hashi_vault_helper.py

```diff
@@ -29,7 +29,6 @@ def vault_token_via_env(vault_token):
         yield
 
 
-@pytest.mark.skipif(sys.version_info < (2, 7), reason="Python 2.7 or higher is required.")
```

Comment:
Don't you somehow need the skip_python fixture here (and in other places)? Or how does that work?

Reply:
@felixfontein yes indeed! It's in an autouse fixture in `tests/unit/conftest.py` (which existed before this, I just never went back to take out this one thing). That fixture is updated in this PR to change the version.